client/src/test/java/io/confluent/kafka/schemaregistry/client/CachedSchemaRegistryClientTest.java

```diff
@@ -87,6 +87,27 @@ public void testHttpHeaderConfiguration() {
     verify(restService);
   }
 
+  @Test
+  public void testDuplicateClientNamespaceConfiguration() {
+    Map<String, String> configs = Collections.singletonMap("key", "value");
+    restService.configure(configs);
+    expectLastCall();
+
+    replay(restService);
+
+    Map<String, String> duplicateConfigs = new HashMap<>();
+    duplicateConfigs.put("key", "value");
+    duplicateConfigs.put("schema.registry.key", "value");
```

Comment:
Don't you want to try with a different "value" to actually see whether toMap mergefunction got executed and indeed replaces a new value?

Reply:
Actually the code `(existing, replacement) -> replacement` isn't really that meaningful, since most of the time the passed in Map will be a HashMap, and the order of traversal is non-deterministic.   I did verify that if a LinkedHashMap with different values is passed, the last value is retained.